cf_xarray/tests/test_accessor.py

```diff
@@ -52,20 +52,28 @@ def test_coordinates():
     assert actual == expected
 
 
-def test_cell_measures():
+def test_cell_measures(capsys):
     ds = airds.copy(deep=True)
     ds["foo"] = xr.DataArray(ds["cell_area"], attrs=dict(standard_name="foo_std_name"))
     ds["air"].attrs["cell_measures"] += " foo_measure: foo"
     assert "foo_std_name" in ds.cf["air_temperature"].cf
 
     ds["air"].attrs["cell_measures"] += " volume: foo"
-    expected = dict(area=["cell_area"], volume=["foo"])
+    expected = dict(area=["cell_area"], foo_measure=["foo"], volume=["foo"])
```

Comment:
can we also test `ds.cf["foo_measure"]`

Reply:
After #117 I think we can allow cell_measures in `ds.cf.keys`? See c45cddc35bcfbcf626ad7af72371e39160eed09b